charts/bulk-scan-orchestrator/requirements.yaml

```diff
@@ -5,4 +5,4 @@ dependencies:
   - name: servicebus
     version: ~0.2.2
     repository: '@hmctspublic'
-    condition: bso.servicebus.enabled
+    condition: servicebus.enabled
```

Comment:
Why are we removing the first component?

Reply:
they are added to make them unique, with new version  "indentation of the condition needs to inside the base chart." we do not need

Reply:
they were initially there so that it is not applicable globally and now since they are inlined we don't them as such